docs/swagger.yaml

```diff
@@ -1984,6 +1993,7 @@ components:
             In other words: This is the URL Vipps sends the Vipps user back to.
             URLs passed to Vipps must validate, see
             [URL validation](https://github.com/vippsas/vipps-ecom-api/blob/master/vipps-ecom-api.md#url-validation).
+            The URL must use HTTPS.
```

Comment:
fallback doesn't require https
its only https if its a web redirect, but native switch like `ruter://` is allowed

Reply:
Thanks, added/fixed.